core/src/test/scala/org/apache/spark/storage/MemoryStoreSuite.scala

```diff
@@ -61,7 +60,7 @@ class MemoryStoreSuite
   }
 
   def makeMemoryStore(maxMem: Long): (MemoryStore, BlockInfoManager) = {
-    val memManager = new StaticMemoryManager(conf, Long.MaxValue, maxMem, numCores = 1)
+    val memManager = new UnifiedMemoryManager(conf, maxMem, (0.4 * maxMem).toInt, 1)
```

Comment:
This mimics the behavior of the old memory manager with spark.storage.unrollFraction=0.4